com.pinegrow.Pinegrow.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.pinegrow.Pinegrow
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: pinegrow
+separate-locales: false
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+modules:
+
+  - name: pinegrow
+    buildsystem: simple
+    build-commands:
+      - install -D apply_extra /app/bin/apply_extra
+      - install -D pinegrow /app/bin/pinegrow
+      - install -Dm644 -t /app/share/metainfo com.pinegrow.Pinegrow.appdata.xml
+      - install -Dm644 -t /app/share/applications com.pinegrow.Pinegrow.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps com.pinegrow.Pinegrow.svg
+      - install -d "${FLATPAK_DEST}/extensions"
+      - install -d "${FLATPAK_DEST}/packages"
```

Comment:
I don't think these are necessary as there is nothing installed there.

Reply:
Fixed, was left from my bootstrap